mkdocs/docs/configuration.md

```diff
@@ -198,19 +198,20 @@ catalog:
-| rest.signing-name   | execute-api                      | The service signing name to use when SigV4 signing a request                                       |
-| oauth2-server-uri   | https://auth-service/cc          | Authentication URL to use for client credentials authentication (default: uri + 'v1/oauth/tokens') |
+| Key                 | Example                          | Description                                                                                                                                                                                             |
+| ------------------- | -------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| uri                 | https://rest-catalog/ws          | URI identifying the REST Server                                                                                                                                                                         |
+| ugi                 | t-1234:secret                    | Hadoop UGI for Hive client.                                                                                                                                                                             |
+| credential          | t-1234:secret                    | Credential to use for OAuth2 credential flow when initializing the catalog                                                                                                                              |
+| token               | FEW23.DFSDF.FSDF                 | Bearer token value to use for `Authorization` header                                                                                                                                                    |
+| scope               | openid offline corpds:ds:profile | Desired scope of the requested security token (default : catalog)                                                                                                                                       |
+| resource            | rest_catalog.iceberg.com         | URI for the target resource or service                                                                                                                                                                  |
+| audience            | rest_catalog                     | Logical name of target resource or service                                                                                                                                                              |
+| access-delegation   | remote-signing                   | A comma-separated list of access mechanisms to signal the server that the client supports delegated access. It will to be sended in `X-Iceberg-Access-Delegation` header. (default: vended-credentials) |
```

Comment:
nit:
```suggestion
| access-delegation   | remote-signing                   | A comma-separated list of access mechanisms to signal the server that the client supports delegated access. It will be sent in `X-Iceberg-Access-Delegation` header. (default: vended-credentials) |
```